00 Base/src/redux/actions/fruitsActions.tsx

```diff
@@ -0,0 +1,28 @@
+import { GET_ALL_FRUITS, DELETE_FRUIT, ADD_FRUIT } from './../constants';
+import { getListOfFruit } from '../../myApi/index';
```

Comment:
Please could you simulate api requests using axios with this fake api: https://jsonplaceholder.typicode.com/?